protocols/gossipsub/src/behaviour.rs

```diff
@@ -527,14 +526,14 @@ where
     /// Returns [`Ok(true)`] if the subscription worked. Returns [`Ok(false)`] if we were already
     /// subscribed.
     pub fn subscribe<H: Hasher>(&mut self, topic: &Topic<H>) -> Result<bool, SubscriptionError> {
-        debug!("Subscribing to topic: {}", topic);
+        tracing::debug!(%topic, "Subscribing to topic");
```

Comment:
//CC @jxs and @AgeManning to make sure you are aware of the switch to `tracing` in libp2p in general and Gossipsub in particular.

Reply:
Thanks for the ping! 

I'm in favour of tracing! I use it in other libraries :)